tests/resources/single_destination.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
eventually may have a lot of testcases for pathfinder, not sure we want a separate json file for each.

in the future maybe consider having 1 json file with all the cases, but have to parse out the separate routes in the 1 testcase file i guess